### DISCUSSION TITLE
Anyone can help me to deploy to Google cloud ?

### DISCUSSION THREAD

**Discussion by Spriteiplan** [saleor/saleor#9563] [Deployment]
HI all, I would like to know what env. we need to do setup on google cloud? Or anyone can help us to setup?

**Comment by karolkielecki** [comment]
Hi @Spriteiplan, Saleor's preferred deployment method is using Docker. The dashboard and the storefront are static applications that do not require additional servers. You can find more info in our docs: https://docs.saleor.io/docs/3.x/developer/running-saleor/docker

**Reply by Spriteiplan** [reply]
Hi @karolkielecki Thanks for yo help. I have one more question about how to connect saleor connect to Google Cloud SQL as I'm stuck here. there are any documents for this integration? Thanks again.

**Reply by karolkielecki** [reply]
I've found an unofficial guide that wasn't approved by any of our team members, but maybe it can be helpful: https://www.thestrugglingdeveloper.com/2021/08/16/saleor-production-deploy-gcp-gce-docker-cloudflare-dns-nginx-gcs-gcp-sql/

**Reply by Spriteiplan** [reply]
@karolkielecki I will check it.  Thank you for your help.

**Reply by Spriteiplan** [reply]
Hi @karolkielecki we still have problem that can't fix.

We run saleor api,dashboard and react storefrontend.
API and dashboard are compatible each other.
And react storefrontend works correctly if connect sample api.
But if connect api that we run, the graphql query of react frontend don't work.

Please suggest me on this issue.
Thank you

**Comment by Spriteiplan** [comment]
Anyone provied deploy service on google cloud? Please let me know. Thanks

**Reply by MaxTeiger** [reply]
Hello @Spriteiplan, did you manage to make it work on GCP ?

**Comment by ruptechnologies** [comment]
Awesome! Deploying Saleor on Google Cloud can be a great way to scale your eCommerce solution. I’ll guide you step-by-step using Google Cloud Compute Engine (GCE) or Google Kubernetes Engine (GKE) — let’s start with the simpler method: GCE with Docker + Docker Compose, which is great for development and staging.


https://cyberian.pk/topic/2729/step-by-step-guide-to-deploy-saleor-on-google-cloud-gce#google_vignette
